eogrow/pipelines/rasterize.py

```diff
@@ -103,12 +103,12 @@ def __init__(self, *args: Any, **kwargs: Any):
 
         self.filename: Optional[str] = None
 
-        if len({c.output_feature[0].is_temporal() for c in self.config.columns}) != 1:
+        if len({self._is_temporal(c.output_feature) for c in self.config.columns}) != 1:
             raise ValueError("All output features have to be either temporal or timeless!")
 
         if isinstance(self.config.vector_input, str):
             self.filename = self._parse_input_file(self.config.vector_input)
-            output_is_temporal = self.config.columns[0].output_feature[0].is_temporal()
+            output_is_temporal = self._is_temporal(self.config.columns[0].output_feature)
```

Comment:
shouldn't this check ALL output features?

Reply:
I agree.

It is partially already done; there is a check for input features to be all temporal or timeless. So the final (missing) check is that output feature also conforms to input feature timeliness.

Reply:
Added the (quick) test, we can address how to check this properly (at schema validation phase?) in another issue.